platform/macos/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Labels written in Arabic, Hebrew, and other scripts are written right-to-left. Arabic characters are correctly shaped. ([#6984](https://github.com/mapbox/mapbox-gl-native/pull/6984), [#7123](https://github.com/mapbox/mapbox-gl-native/pull/7123))
 * Improved the line wrapping behavior of point-placed labels written in Chinese, Japanese, and Yi. ([#6828](https://github.com/mapbox/mapbox-gl-native/pull/6828))
 * Fixed an issue where translucent point annotations along tile boundaries would be drawn darker than expected. ([#6832](https://github.com/mapbox/mapbox-gl-native/pull/6832))
+* Added the Simplified Chinese localization. ([#7316](https://github.com/mapbox/mapbox-gl-native/pull/7316))
```

Comment:
I just realized this PR doesn’t actually localize the macOS SDK. It could – you’d add the localization to macos.xcodeproj, reuse platform/darwin/resources/zh-Hans.lproj/Foundation.strings, and add platform/macos/resources/zh-Hans.lproj/Localizable.strings ([just 10 easy strings](https://github.com/mapbox/mapbox-gl-native/blob/3d2dc471918a38610105509a73eda39d5112ce6b/platform/macos/sdk/Base.lproj/Localizable.strings)). But we can take care of that in a separate PR. Would you mind removing this changelog entry in the meantime?